runtime/jvmti/jvmtiStartup.c

```diff
@@ -1074,8 +1242,11 @@ static BOOLEAN
 isAgentLibraryLoaded(J9JavaVM *vm, const char *library, BOOLEAN decorate)
 {
 	BOOLEAN result = FALSE;
+	/* the library is a platform independent name */
```

Comment:
This comment is not about `libraryLen`; please move it back where it was (after `if (decorate)`).

Reply:
Restored the comment to the previous location.